File: typescript/8kyu/Vowel_Count.ts
export class Kata {
  static getCount(str: string): number {
    let total = 0;
    for (let letter of str.split("")) {
      if (["a", "e", "i", "o", "u"].some((el) => el === letter)) total++;
    }
    return total;
  }
}

File: python/7kyu/Vowel_Count.py
def get_count(sentence):
    vowels = ["a", "e", "i", "o", "u"]
    total = 0
    Sentence_arr = list(sentence)
    for letter in Sentence_arr:
        for vowel in vowels:
            if vowel == letter:
                total += 1
    return total